experimental/web_dashboard/pubspec.yaml

```diff
@@ -7,16 +7,16 @@ environment:
   sdk: ^3.1.0
 
 dependencies:
-  charts_flutter: ^0.12.0
   cloud_firestore: ^4.0.1
   collection: ^1.16.0
+  community_charts_flutter: ^1.0.2
   cupertino_icons: ^1.0.0
   firebase_auth: ^4.0.1
   firebase_core: ^2.0.0
   flutter:
     sdk: flutter
   google_sign_in: ^6.0.0
-  intl: ^0.17.0
+  intl: any # Pinned by Flutter SDK version
```

Comment:
Is this now the recommended practice?

Reply:
It's the only way I can see to keep the project compiling under multiple SDK versionss with different `intl` versions pinned. The other option is to successively comment out `web_dashboard` as the version conflict wanders down the `master` -> `beta` -> `stable` cascade.

I can't remember where I saw this previously, this isn't my creation =)